EN/02-docker/installing-docker.adoc

```diff
@@ -0,0 +1,85 @@
+Installing Docker
```

Comment:
I've an idea, maybe we should just merge the Installation/Docker(recommended) in this page. And for Installation->Mac/Linux/Windows section we introduce in bold :
**"We recommend you to try Feel++ using Docker (see docker section +link) before going further in Feel++ installation"**.

What do you think ? (That way we don't have kind of phantom section :) )